chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -47,7 +47,7 @@
 
   bash "disable_openstack_services" do
     code <<-EOF
-      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1) \
+      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1 | grep -Ev 'keystone|neutron') \
```

Comment:
we are going to keep `keystone` and `neutron` up. It was to keep in line semantically __service disabled__ with __service_stopped__. I guess it may be a problem after the node reboots and tries to bring up these services while we upgrade and `chef-client` is down. So I'm removing those exceptions.